src/components/help/HelpItemFromFile.js

```diff
@@ -0,0 +1,33 @@
+import HelpItem from "@/components/help/HelpItem";
+import Markdown from "react-markdown";
+import rehypeRaw from "rehype-raw";
+import React from "react";
+import usePublicFile from "@/state/query-hooks/use-public-file";
+import Loading from "@/components/util/Loading";
+
+export default function HelpItemFromFile({ item, key }) {
+  const { data, isLoading, isError } = usePublicFile(item.filePath);
+
+  const body = () => {
+    if (isLoading) {
+      return <Loading>Loading...</Loading>;
+    }
+    if (isError) {
+      return <div>Sorry, there was an error loading this help.</div>;
+    }
+    return <Markdown rehypePlugins={[rehypeRaw]}>{data ?? item.body}</Markdown>;
+  };
+
+  return (
+    <HelpItem
+      key={key}
```

Comment:
I think this key throws a warning in the console as it isn't needed in this context.

The warning in question:
```
Warning: HelpItemFromFile: `key` is not a prop. Trying to access it will result in `undefined` being returned. If you need to access the same value within the child component, you should pass it as a different prop. (https://reactjs.org/link/special-props)
```